### ISSUE TITLE
docker image deploy does not work

### ISSUE THREAD

**Issue by debianmaster** [cloudfoundry/cf-for-k8s#120]
```
cf push test-app -o cloudfoundry/diego-docker-app
Using docker repository password from environment variable CF_DOCKER_PASSWORD.
Pushing app test-app to org test-org / space test-space as admin...
Getting app info...
Updating app with these attributes...
  name:                test-app
  docker image:        cloudfoundry/diego-docker-app
  disk quota:          1G
  health check type:   port
  instances:           1
  memory:              1G
  stack:               cflinuxfs3
  routes:
    test-app.cf.rsc7.ae

Updating app test-app...
Mapping routes...

Stopping app...

Waiting for app to start...

Stats unavailable: Stats server temporarily unavailable.
FAILED```


```
k get deploy,po,virtualservice -A | grep test-app
cf-workloads           pod/test-app-test-space-7191bc1999-0                               2/2     Running      0          2m46s```

**Comment by debianmaster** [comment]
more debug info

```
cf-blobstore           cf-blobstore-minio-67949cc577-hgr9h                            2/2     Running      0          24m
cf-db                  cf-db-postgresql-0                                             2/2     Running      0          24m
cf-system              capi-api-server-7d4f95b77-pt6qm                                4/4     Running      2          24m
cf-system              capi-api-server-7d4f95b77-rp4bs                                4/4     Running      2          24m
cf-system              capi-clock-87b479dc4-fb8gj                                     2/2     Running      3          24m
cf-system              capi-deployment-updater-5f47c778b7-jvdrl                       2/2     Running      4          24m
cf-system              capi-kpack-watcher-cdd57c997-dflrv                             2/2     Running      0          24m
cf-system              capi-worker-677567fc89-mrfj7                                   2/2     Running      4          24m
cf-system              cfroutesync-69ff687868-lmnmv                                   2/2     Running      0          24m
cf-system              eirini-76999588fd-tvxxr                                        2/2     Running      0          24m
cf-system              fluentd-cwlqb                                                  2/2     Running      4          24m
cf-system              fluentd-lfk8k                                                  2/2     Running      4          24m
cf-system              fluentd-mhdf2                                                  2/2     Running      4          24m
cf-system              fluentd-vt6r6                                                  2/2     Running      4          24m
cf-system              log-cache-78b5776bdc-xkh48                                     5/5     Running      5          24m
cf-system              metric-proxy-55c449fcb6-7xhrf                                  2/2     Running      0          24m
cf-system              uaa-75fcd69d58-xpwxx                                           2/2     Running      3          24m
cf-workloads-staging   14c49278-4920-4975-a8fe-c56087809924-build-1-jqgfp-build-pod   0/1     Init:Error   0          10m
cf-workloads-staging   50ed589a-43c7-4177-9bfc-4d6dbc05691b-build-1-7n4s6-build-pod   0/1     Init:Error   0          11m
cf-workloads-staging   751f407f-2534-4478-8bc0-149586d4852a-build-1-vht45-build-pod   0/1     Init:Error   0          14m
cf-workloads           test-app-test-space-7191bc1999-0                               2/2     Running      0          3m26s
istio-system           istio-citadel-784f4d84d5-wctw6                                 1/1     Running      0          25m
istio-system           istio-galley-556477b64c-q866j                                  2/2     Running      0          25m
istio-system           istio-ingressgateway-4cc2g                                     2/2     Running      0          24m
istio-system           istio-ingressgateway-jvdnm                                     2/2     Running      0          24m
istio-system           istio-ingressgateway-ld9jp                                     2/2     Running      0          24m
istio-system           istio-ingressgateway-x6wwb                                     2/2     Running      0          24m
istio-system           istio-pilot-69b9994b9d-cl4jw                                   2/2     Running      0          25m
istio-system           istio-policy-7d6b4b694d-pthpr                                  2/2     Running      1          25m
istio-system           istio-sidecar-injector-dd4c47b7-66tm6                          1/1     Running      0          25m
istio-system           istio-telemetry-7b67ff44f5-gwvjh                               2/2     Running      1          25m
kpack                  kpack-controller-58544584f4-s4dht                              1/1     Running      0          25m
kpack                  kpack-webhook-74fd57db98-jr2lx                                 1/1     Running      0          25m```

**Comment by acosta11** [comment]
Hi @debianmaster ,

Do you get any additional logs from `cf logs test-app --recent` to use for troubleshooting? And if not, do any of the failed build pods in the cf-workloads-staging namespace indicate which of the build steps failed? We have a new debugging doc that captured two common issues we've encountered with failed builds so far: https://github.com/cloudfoundry/cf-for-k8s/blob/master/docs/debugging.md .

Thanks,
Andrew

**Comment by acosta11** [comment]
Potentially the same fix as suggested in #121 .

Are you using `https://index.docker.io/v1/` for the docker hostname like the example values file?

Thanks,
Andrew

**Comment by davewalter** [comment]
Hi @debianmaster,

We're going to go ahead and close this issue. Please feel free to reopen it if the above suggestion does not solve your problem.

Thanks,
Dave and @jamespollard8
